lib/sidekiq/cli.rb

```diff
@@ -76,7 +76,7 @@ def run
       # Since the user can pass us a connection pool explicitly in the initializer, we
       # need to verify the size is large enough or else Sidekiq's performance is dramatically slowed.
       cursize = Sidekiq.redis_pool.size
-      needed = Sidekiq.options[:concurrency] + 2
+      needed = Sidekiq.options[:concurrency] + Sidekiq::RedisConnection::AUXILIARY_CONNECTIONS
```

Comment:
Here we are in the lexical scope of the `Sidekiq` module, so constant lookup will find `RedisConnection` as far as `sidekiq.rb` file already required `redis_connection.rb` globally. IMO, this is more appropriate:
```
require 'sidekiq/redis_connection'
...
needed = Sidekiq.options[:concurrency] + RedisConnection::AUXILIARY_CONNECTIONS
```

Reply:
👍 updated!